src/controllers/auth.controller.js

```diff
@@ -0,0 +1,27 @@
+const authService = require('../services/auth.service');
+
+const signUp = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    await authService.signUp(email, password);
+
+    res.status(201).end();
+  } catch (err) {
+    res.status(err.statusCode || 400).json({ message: err.message });
+  }
+};
+
+const signIn = async (req, res) => {
+  try {
+    const { email, password } = req.body;
```

Comment:
email, password 데이터 중 하나라도 요청 데이터에 포함되지 않았을 경우 예외처리를 해주면 좋을 것 같습니다!
```suggestion
    const { email, password } = req.body;
    
    if ( !email || !password) {
      throw new Error("key error")
    }
```

Reply:
해당 부분은 반영이 안되어 있습니다! 반영해주세요 승현님~!